templates/CRM/common/customDataBlock.tpl

```diff
@@ -1,19 +1,17 @@
+
 {if !empty($customDataType)}
   <div id="customData"></div>
   {*include custom data js file*}
   {include file="CRM/common/customData.tpl"}
-  {assign var='cid' value=$cid|default:'false'}
   {literal}
   <script type="text/javascript">
     CRM.$(function($) {
       {/literal}
-      {if !empty($customDataSubType)}
-        CRM.buildCustomData('{$customDataType}', {$customDataSubType}, false, false, false, false, false, {$cid});
-      {else}
-        CRM.buildCustomData('{$customDataType}', false, false, false, false, false, false, {$cid});
-      {/if}
-      {literal}
+      CRM.buildCustomData('{$customDataType}', {$customDataSubType|default:false}, false, false, false, false, false, {$cid});
```

Comment:
@eileenmcnaughton are we sure that $cid won't have been defaulted to false by smarty

Reply:
@seamuslee001 oh so you are suggesting that it is going from `'false'` to `false` ?

Reply:
@seamuslee001 ok - I think it's a little better now